TikTokApi/tiktok.py

```diff
@@ -216,49 +216,79 @@ def handle_request(request):
         self.sessions.append(session)
         await self.__set_session_params(session)
 
-    async def create_sessions(
+    async def __create_authenticated_session(
         self,
+        auth_cookies: str,
+        url: str = "https://www.tiktok.com",
+        proxy: str = None,
+        context_options: dict = {},
+        sleep_after: int = 1,
+        suppress_resource_load_types: list[str] = None,
+        timeout: int = 30000,
+    ):
+        """Create an authenticated TikTokPlaywrightSession using provided cookies
+
+        Args:
+            auth_cookies (str): Authentication cookies string from TikTok in the format "key1=value1;key2=value2"
+            url (str): The url to start the session on
+            proxy (str): Proxy to use for the session
+            context_options (dict): Additional options for the playwright context
+            sleep_after (int): Time to sleep after creating the session
+            suppress_resource_load_types (list[str]): Types of resources to suppress
+            timeout (int): Navigation timeout in milliseconds
+
+        Returns:
+            None
+        """
+        # Parse cookies string into a dictionary
+        cookies = {}
+        for cookie in auth_cookies.split(';'):
```

Comment:
Since users can attach `cookies` in the current code I think users can just do this on their side before calling `create_sessions`. If there's some other justification that this should be in the library let me know!